spec/jobs/create_case_comment_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../spec_helper'
+
+RSpec.describe Jobs::CreateCaseComment do
+  include_context "spec helper"
+
+  fab!(:topic) { Fabricate(:topic) }
+  fab!(:_case) { Fabricate(:salesforce_case, topic: topic) }
+  fab!(:post) { Fabricate(:post, topic: topic) }
+
+  it 'creates a case comment object on Salesforce' do
+    topic.custom_fields["has_salesforce_case"] = true
+    topic.save!
+
+    ::Salesforce::CaseComment.any_instance.expects(:create!).once
```

Comment:
We usually want to avoid stubbing. Would it be possible to test that the `SalesForce::CaseComment` record has been created instead?

Reply:
We already doing it in the [`spec/models/case_comment_spec.rb`](https://github.com/discourse/discourse-salesforce/pull/4/files#diff-aa551af8ec9e47d2e792746274f8faae1a4c40efd752ce09704624a107f7b937) file. That's I skipped here.